### ISSUE TITLE
Wrong rank,rating, camera settings, etc

### ISSUE THREAD

**Issue by Mannichon** [rocket-league-replays/rocket-league-replays#39]
So, when i first started using the site and uploaded a bunch of replays it was super accurate, except about my camera settings but no big deal. I havent uploaded much since, havent even saved replays in awhile, though i did save a bunch that i didnt upload. Then today i uploaded all the ones i had saved, and started saving some more. When i checked the website to get the link to share a replay, i noticed the replay had a much much higher rank than i had/have and i blew it off figuring it was an unranked replay and that i had played with a higher ranked teammate. But from there I started checking, and ALOT of my replays are either not given a rank, or given a much much higher rank than i have earned. And these are competative ranked matches, not hoops or rumble or something. Basically all the stats the site shows are wrong, but rocketleaguestats are tracking me properly.

**Comment by danielsamuels** [comment]
Yeah, you can't trust the rankings on the site unfortunately. I've been waiting on Psyonix to do some stuff on their end, but nothing has happened yet.

**Comment by tfausak** [comment]
Unfortunately rank information isn't stored in the replays. Camera settings are, though. Do you have a specific example of some bad camera settings?

**Comment by danielsamuels** [comment]
Ranks should be fixed now. Camera settings are something I need to work on.

**Comment by tfausak** [comment]
I recently discovered that there is a bug with camera settings in replays. Sometimes they just aren't replicated, so they show up as the default instead of whatever the player was using. See this Reddit thread for example: <https://www.reddit.com/r/RocketLeague/comments/4yio7m/bug_wide_known_default_camera_glitch/>.

**Comment by danielsamuels** [comment]
Yeah, the main issue is that they're simply stored in a different way / place in the JSON now, I just need to update my parser to take that into account.
